notebooks/persistence_example.ipynb

```diff
@@ -23,9 +23,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Pickle\n",
+    "## `torch.save` and `torch.load`\n",
+    "\n",
+    "The most general-purpose way of saving TextWiser objects is to use [torch.save](https://pytorch.org/tutorials/beginner/saving_loading_models.html#save-load-entire-model). This has the benefit of letting all Torch tensors be handled by PyTorch, while pickling everything else.\n",
```

Comment:
So the change for GPU vs CPU persistence is just to use a different external-to-textwiser protocol?

Reply:
Yup! There wasn't an explicit `save` function for the TextWiser object anyway, and these new functions reuse the existing functionality that enables `load_state_dict` and `save_state_dict` to work while being simpler to use.